QModManager/API/ModLoading/Internal/QMod.cs

```diff
@@ -0,0 +1,70 @@
+﻿namespace QModManager.API.ModLoading.Internal
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Reflection;
+
+    internal class QMod
```

Comment:
Rename this to something else so we it doesn't get confused with the other `QMod` class

Reply:
My intention is that all these classes can combine into one, or at least share a common interface.  
That I have yet to dig into.

Reply:
Still WiP

Reply:
We are back to a single internal `QMod` class.  
There is a `IQMod` interface for public use.  
Other interfaces implemented by this class are for internal use only.